p2p/antiflood/factory/disabledAntiFlood.go

```diff
@@ -0,0 +1,31 @@
+package factory
+
+import (
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+type disabledAntiFlood struct {
```

Comment:
should add at least one test that call all method and a defer func that, if recovered from panic, will call assert.Fail. In this way we keep our code coverage percentage high and assure 100% that the code will never panic

Reply:
ok. done